client/orm/orm_object.go

```diff
@@ -15,12 +15,14 @@
 package orm
 
 import (
+	"context"
 	"fmt"
 	"reflect"
 )
 
 // an insert queryer struct
 type insertSet struct {
+	ctx    context.Context
```

Comment:
it's similar to QuerySter, and I think we could remove this field and trigger it like other interface.

Reply:
done